iox_client_management.go

```diff
@@ -44,3 +45,17 @@ func (c *Client) GetServerStatus(ctx context.Context) (string, error) {
 	}
 	return "?", nil
 }
+
+func (c *Client) Delete(ctx context.Context, databaseName, tableName string, startTime, stopTime time.Time, predicate string) error {
```

Comment:
👍 

I do wonder how we will test code in this repo, but we can sort that out in a bit

Reply:
I'm thinking we run an instance of IOx, but waiting for an idea about how to do that most efficiently.